<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemultStreamDialog</class>
 <widget class="QDialog" name="DemultStreamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="outputLB">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outputLED">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outputPBN">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="channelSB"/>
       </item>
       <item>
        <widget class="QSlider" name="channelSL">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="infoLB">
       <property name="text">
        <string>Выберите канал для демультеплексирования потока(нумерация с 1) и файл, куда будет записан результат. Данная функция демультеплексирует выбранный канал из потока E1(период 256) с учетом разворота бит(преобразования из сетевого формата в компьютерный). Если открыть полученный файл с частотой дисркетизации 8кГц и 8-битными квантованием, а также применить компандирование с A-законом, то можно получить информацию.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>channelSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channelSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channelSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channelSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DemultStreamDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DemultStreamDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
